pkg/client/injection/reconciler/pipeline/v1alpha1/run/reconciler.go

```diff
@@ -181,7 +181,7 @@ func (r *reconcilerImpl) Reconcile(ctx context.Context, key string) error {
 	// Initialize the reconciler state. This will convert the namespace/name
 	// string into a distinct namespace and name, determine if this instance of
 	// the reconciler is the leader, and any additional interfaces implemented
-	// by the reconciler. Returns an error is the resource key is invalid.
+	// by the reconciler. Returns an error if the resource key is invalid.
```

Comment:
Please do not change generated files.
This file is generated from a template, so the template should be fixed instead.

Reply:
Sorry, I am new to Tekton not very familiar about the codes.
I removed the modified generated codes, maybe I can fix it when I am familiar enough about the template, please have a look again, thanks~